### PR TITLE
Improve README/CONTRIBUTING. Closes #159

### LINKED ISSUE #159: Tidy up README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by brendo**
This is just a minor tidy-up to split the README for consumers, and CONTRIBUTING for contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,60 +6,51 @@
...
+import ReactDOM from 'react-dom'
+import Lincoln from '../Lincoln'
+
+ReactDOM.render(
+  <Lincoln definitionUrl='https://temando.github.io/open-api-renderer/petstore-open-api-v3.0.0-RC2.json' />,
+  document.body
+)
 ```
 
 Alternatively, you can pass the contents of the definition directory to Lincoln:
```
Comment (nfour) [inline README.md]: *directly

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,9 +2,21 @@
```
Comment (nfour) [inline CONTRIBUTING.md:2]: Good stuff

File: README.md [hunk 3]
```diff
@@ -6,60 +6,51 @@
...
 
 To use Lincoln in your React project:
 
 ```js
-import Lincoln from 'react-lincoln'
-
-<Lincoln definitionUrl='https://temando.github.io/open-api-renderer/petstore-open-api-v3.0.0-RC1.json' />
+import React from 'react'
+import ReactDOM from 'react-dom'
+import Lincoln from '../Lincoln'
```
Comment (nfour) [inline README.md]: Shouldn't that be `react-lincoln`?

File: README.md [hunk 4]
```diff
@@ -6,60 +6,51 @@
 [![MIT License](https://img.shields.io/github/license/temando/open-api-renderer.svg)](https://en.wikipedia.org/wiki/MIT_License)
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
```
Comment (nfour) [inline README.md:8]: Add npm badge too?

```md
[![NPM](https://img.shields.io/npm/v/react-lincoln.svg)](https://npmjs.org/packages/react-lincoln/)
```

Also may want to grab that package name on npm asap.

**Review by nfour — CHANGES_REQUESTED** [review]
Nice stuff. Few bits.

File: README.md (hunk 3 above)
Comment (brendo) [reply README.md]: Yep!

File: README.md (hunk 1 above)
Comment (brendo) [reply README.md]: ty

File: README.md (hunk 4 above)
Comment (brendo) [reply README.md:8]: Done, will nab package name tonight/tomorrow.

**Review by nfour — APPROVED** [review]

**Pull request merged**